components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java

```diff
@@ -1066,6 +1066,7 @@ public void updateResidentIdP(IdentityProvider identityProvider, String tenantDo
      * @return Identity Provider's Basic Information array {@link IdpSearchResult}.
      * @throws IdentityProviderManagementServerException server related error while getting Identity  Providers object.
      * @throws IdentityProviderManagementClientException client related error while getting Identity  Providers object.
+     *  @deprecated use {@link #getIdPs(Integer, Integer, String, String, String, String, List)}
      */
     @Override
     public IdpSearchResult getIdPs(Integer limit, Integer offset, String filter, String sortOrder, String sortBy,
```

Comment:
call the new getIdPs(Integer, Integer, String, String, String, String, List) from within this deprecated method